content/en/docs/concepts/services-networking/service.md

```diff
@@ -361,6 +361,10 @@ You can also set the maximum session sticky time by setting
 `service.spec.sessionAffinityConfig.clientIP.timeoutSeconds` appropriately.
 (the default value is 10800, which works out to be 3 hours).
 
+{{< note >}}
+Setting the maximum session sticky time for services that run on Windows is not supported.
```

Comment:
```suggestion
Setting the maximum session sticky time for Services that run on Windows is not supported.
```
What does it mean for a Service to “run on” Windows. For example, if I run two Deployments and select both deployments with a single Service, what happens?
I'm guessing it's undefined behavior, but maybe I'm wrong?

Reply:
Good question and I was struggling with the wording here.
Services are not tied to any OS. 
Does something like:

Setting the maximum session sticky time for services connected to pods running on Windows nodes is not supported?

Unfortunately, I'm not sure exactly what does not work in this scenario and was copying this from other parts of the docs :(

Reply:
I fixed the capitalization. Leaving this comment open for the additional discussions.